### PR TITLE
[Sowmya|Tanya] | Adds logic to fetch drug chart data | A-1204950368410820

### PR DESCRIPTION
**Pull request by tanyaa-tw**

### CODE REVIEW

**Review by Arjun-Go — APPROVED** [review]

File: micro-frontends/src/ipd/IpdDashboard.jsx [hunk 1]
```diff
@@ -19,11 +18,7 @@ export function IpdDashboard(props) {
...
   hostData: PropTypes.shape({
-    patient: PropTypes.shape({
-      uuid: PropTypes.string.isRequired,
-    }).isRequired,
-  }),
-  hostApi: PropTypes.shape({
-    onConfirm: PropTypes.func,
-  }),
+    patientId: PropTypes.string,
+    forDate: PropTypes.instanceOf(Date),
```
Comment (Phanindra-tw) [inline micro-frontends/src/ipd/IpdDashboard.jsx:22]: Do we need to send the forDate from Bahmni Apps?
I think the navigation of dates will be driven from IPD module only. So I think sending only patientId would be sufficient.

**Comment by Arjun-Go** [comment]
Since you don't need the confirmBox in the patientDasboardController, the mocks for the confirmbox can be removed from the test file as well. - https://github.com/Bahmni/openmrs-module-bahmniapps/blob/01b30fcbb51e2d39f048473a8be709f7198d1966/ui/test/unit/clinical/controllers/patientDashboardController.spec.js#L46

File: ui/app/clinical/dashboard/controllers/patientDashboardController.js [hunk 2]
```diff
@@ -25,29 +25,10 @@ angular.module('bahmni.clinical')
...
-                        };
-
-                        confirmBox({
-                            scope: dialogScope,
-                            actions: [{ name: "okay", display: "Okay" }],
-                            className: "ngdialog-theme-default"
-                        });
-                    }
+                    patientId: $stateParams.patientUuid,
+                    forDate: new Date().toUTCString()
```
Comment (Phanindra-tw) [inline ui/app/clinical/dashboard/controllers/patientDashboardController.js:31]: Same as abve

Comment (tanyaa-tw) [reply ui/app/clinical/dashboard/controllers/patientDashboardController.js:31]: We are directly passing the epoch value of date from bahmniapps. In IPD-frontend repo we are directly receiving the epoch value for forDate

Comment (Phanindra-tw) [reply ui/app/clinical/dashboard/controllers/patientDashboardController.js:31]: Okay, for now lets keep it like this, We might have to revisit this when navigation card is played.

**Review by Phanindra-tw — APPROVED** [review]

**Pull request merged**